db/migrate/20210125135243_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :track_name
+      t.string :artists
+      t.text :image
```

Comment:
is the image a text because you get the image as a huge string?

Reply:
Yes, due to there is a possibility that is can be a long string